<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="tit_transportadora">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>370</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Extra Bold</family>
      <pointsize>13</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>TRANSPORTADORA</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>84</y>
      <width>630</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <widget class="QLineEdit" name="mcgc">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>140</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMask">
       <string>99.999.999/9999-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="mrazao">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>100</y>
        <width>391</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Código Transportadora:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mcpf">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>220</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMask">
       <string>999.999.999-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>220</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CPF:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="mdata_cad">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>20</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Inscrição Estadual:*</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mcod_tran">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>60</y>
        <width>61</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="minsc">
      <property name="geometry">
       <rect>
        <x>209</x>
        <y>180</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Razão Social:*</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Data de Cadastro:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CNPJ:*</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>260</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Observação:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mobs">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>391</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>60</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Apolice Seguro:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Placa do carro:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mplaca">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="mantt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="mapolice">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>420</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>UF:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>180</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ANTT:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>240</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Endereco</string>
     </attribute>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>260</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Telefone 1:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>251</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>300</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Telefone 2:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>250</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>220</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Celular:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mbairro">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>20</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="mcep">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMask">
       <string>99999-999</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>20</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Endereço:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>180</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>CEP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>140</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>UF:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mtel1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="mtel2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>140</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Cidade:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mendereco">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>391</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>35</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>60</y>
        <width>100</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Bairro:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mfax">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>14</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Cobranca</string>
     </attribute>
     <widget class="QLineEdit" name="mct_cobran">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>220</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Vendedor:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>140</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Contato Faturamento:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>180</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Contato Cobranca:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mct_fatura">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>20</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Tipo de Cobranca:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mct_gerente">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="mct_vendedo">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>127</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>60</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Prazo de Pagamento</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>100</y>
        <width>140</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Contato Gerente:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_banco">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>89</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Banco</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_carteira">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>89</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Carteira</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mprazo_pag">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="inputMask">
       <string>9-99</string>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>SAIR.png</normaloff>SAIR.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_salvar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;file:///C:/Users/mebbc/OneDrive/Imagens/salvar.qrc.png&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>SALVAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>SALVAR.png</normaloff>SALVAR.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>HTI.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>mdata_cad</tabstop>
  <tabstop>mcod_tran</tabstop>
  <tabstop>mrazao</tabstop>
  <tabstop>mcgc</tabstop>
  <tabstop>mcpf</tabstop>
  <tabstop>minsc</tabstop>
  <tabstop>mobs</tabstop>
  <tabstop>mplaca</tabstop>
  <tabstop>mantt</tabstop>
  <tabstop>mapolice</tabstop>
  <tabstop>mendereco</tabstop>
  <tabstop>mbairro</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>mcep</tabstop>
  <tabstop>mtel1</tabstop>
  <tabstop>mtel2</tabstop>
  <tabstop>mfax</tabstop>
  <tabstop>rb_banco</tabstop>
  <tabstop>rb_carteira</tabstop>
  <tabstop>mprazo_pag</tabstop>
  <tabstop>mct_gerente</tabstop>
  <tabstop>mct_fatura</tabstop>
  <tabstop>mct_cobran</tabstop>
  <tabstop>mct_vendedo</tabstop>
  <tabstop>bt_salvar</tabstop>
  <tabstop>bt_sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
